src/main/java/pl/touk/sputnik/processor/detekt/DetektProcessor.java

```diff
@@ -79,9 +82,19 @@ private Detektor buildDetector(String commonPath) {
         } else {
             config = loadDefaultConfig();
         }
-        ProcessingSettings processingSettings = new ProcessingSettings(FileSystems.getDefault().getPath(commonPath), config, new ArrayList<PathFilter>(), false, false, new ArrayList<Path>());
-
-        return DetektFacade.INSTANCE.instance(processingSettings, new RuleSetLocator(processingSettings).load(), Arrays.asList(new LoggingFileProcessor()));
+        ProcessingSettings processingSettings = new ProcessingSettings(
+                FileSystems.getDefault().getPath(commonPath),
+                config,
+                new ArrayList<PathFilter>(),
+                false,
+                false,
+                new ArrayList<Path>(),
+                executor,
+                System.out,
+                System.err
```

Comment:
Why should we pass these streams here? What are they for? Shouldn't Sputnik gather this output and tunnel it somewhere to comments?

Reply:
I really don't know and I am not sure that i would like to know...
I will add dedicated stream